content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,494 @@
+```
+
+Quick test:
+
+```shell
+sudo systemctl is-active kubelet.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -u kubelet.service
+```
+
+Query the Kubelet's API endpoint at `http://localhost:10255/pods`:
+
+```shell
+curl http://localhost:10255/pods | jq '.'
+```
+
+The output is similar to:
+
+```json
+{
+  "kind": "PodList",
+  "apiVersion": "v1",
+  "metadata": {},
+  "items": null
+}
+```
+
+## Run a Pod in the Kubelet
+
+In standalone mode, you can run Pods using Pod manifests. The manifests can either
+be on the local filesystem, or fetched via HTTP from a configuration source.
+
+Create a manifest for a Pod:
+
+```shell
+cat <<EOF > static-web.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+EOF
+```
+
+Copy the `static-web.yaml` manifest file to the `/etc/kubernetes/manifests` directory.
+
+```shell
+sudo cp static-web.yaml /etc/kubernetes/manifests/
+```
+
+### Find out information about the Kubelet and the Pod {#find-out-information}
+
+The Pod networking plugin creates a network bridge (`cni0`) and a pair of `veth` interfaces
+for each Pod (one of the pair is inside the newly made Pod, and the other is at the host level).
+
+Query the Kubelet's API endpoint at `http://localhost:10255/pods`:
+
+```shell
+curl http://localhost:10255/pods | jq '.'
+```
+
+To obtain the IP address of the `static-web` Pod:
+
+```shell
+curl http://localhost:10255/pods | jq '.items[].status.podIP'
+```
+
+The output is similar to:
+
+```
+"10.85.0.4"
+```
+
+Connect to the `nginx` server Pod on `http://<IP>:<Port>` (port 80 is the default), in this case:
+
+```shell
+curl http://10.85.0.4
+```
+
+The output is similar to:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+...
+```
+
+## Where to look for more details
+
+If you need to diagnose a problem getting this tutorial to work, you can look
+within the following directories for monitoring and troubleshooting:
+
+```
+/var/lib/cni
+/var/lib/containers
+/var/lib/kubelet
+
+/var/log/containers
+/var/log/pods
+```
+
+## Clean up
+
+### Kubelet
+
+```shell
+sudo systemctl disable --now kubelet.service
+sudo systemctl daemon-reload
+sudo rm /etc/systemd/system/kubelet.service
+sudo rm /usr/bin/kubelet
+sudo rm -rf /etc/kubernetes
+sudo rm -rf /var/lib/kubelet
+sudo rm -rf /var/log/containers
+sudo rm -rf /var/log/pods
+```
+
+### Container Runtime
+
+```shell
+sudo systemctl disable --now crio.service
+sudo systemctl daemon-reload
+sudo rm -rf /usr/local/bin
+sudo rm -rf /usr/local/lib
+sudo rm -rf /usr/local/share
+sudo rm -rf /usr/libexec/crio
+sudo rm -rf /etc/crio
+sudo rm -rf /etc/containers
+```
+
+### Network Plugins
+
+```shell
+sudo rm -rf /opt/cni
+sudo rm -rf /etc/cni
+sudo rm -rf /var/lib/cni
+```
+
+## Conclusion
```

Comment:
some things that is worth mentioning in this doc is what features are working for standalone mode.

Some examples on what needs to be working off the top of my head:
- Credentials provider for image pull secrets
- Configuring security like AppArmor and SeLinux 
- Host path, emptyDIr
- Probes

Some things worth to mention will not work:
- Any of Kubectl commands with no way to enable those
- PVC
- Runtime class likely not supported

Would be great to have lists like this

Reply:
They seem like valid points to me but they've suggested me to keep this tutorial focused on the objectives, so I need to think how to add these lists and not overly complicate it. 

@sftim, I'll need a little guidance with this last request. Thanks.

Reply:
For guidance, remember you can ask in #sig-docs on Kubernetes' Slack.

> * Credentials provider for image pull secrets
> 
> * Configuring security like AppArmor and SeLinux
> 
> * Host path, emptyDIr
> 
> * Probes

I think we can skip these / I don't see why we need to mention them in this tutorial.

Reply:
Agree